catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/query/OpenSearchQuery.java

```diff
@@ -89,26 +84,16 @@
      * Creates an Implementation of a DDF Query interface. This object is passed from the endpoint
      * to DDF and will be used by sites to perform queries on their respective systems.
      *
-     * @param user
-     *            Credentials of the user performing the query
-     * @param startIndex
-     *            Offset of the returned results.
-     * @param count
-     *            Number of results to return.
-     * @param sortField
-     *            Area that the results should be sorted by. Possible values: 'date' and 'relevance'
-     * @param sortOrderIn
-     *            Order of the results. Possible values: 'asc', 'desc'
-     * @param maxTimeout
-     *            Maximum amount of time for a query to respond.
+     * @param startIndex    Offset of the returned results.
```

Comment:
✏️  Something is still wrong with the formatting... There shouldn't be any spaces between the parameter names and descriptions.